### DISCUSSION TITLE
Light Off Delay for the bottom layers

### DISCUSSION THREAD

**Discussion by Omen2005** [sn4k3/UVtools#565] [Q&A]
Hi!
In the pictures I can see in the wiki, they show that I can set the "light off delay" on the bottom layers before the cure, I want to do that, but I can't find that settings in UVTools.

I have checked tools - edit print parameters and file - Settings - automation but nothing there.. 

After reading this post, I want to try to edit the light off delay before cure to minimize elephant foot 
https://blog.honzamrazek.cz/2022/01/prints-not-sticking-to-the-build-plate-layer-separation-rough-surface-on-a-resin-printer-resin-viscosity-the-common-denominator/

Info:
Anycubic Photon X 6K
Slicer Lychee

**Comment by sn4k3** [comment]
"Settings - automation" is no longer a thing. Use the new methods: https://github.com/sn4k3/UVtools/discussions/534

**Reply by Omen2005** [reply]
Ahh okej then I must have misunderstood some things I read 😆 

In the beginning of the post u linked, it says "It won't work for Anycubic nor Creality Halot printers since there are only 1 global wait time property to set all layers" 

So the new method doesn't work for my anycubic photon?

**Reply by sn4k3** [reply]
Either old or new method wont work, crap firmwares make that impossible.

**Reply by Omen2005** [reply]
That's sucks 😔 so I can't do nothing about the elephant foot? Or you got any other tips?

**Reply by sn4k3** [reply]
You can use the erode method.
Check: https://github.com/sn4k3/UVtools/wiki/Calibration:-Elephant-Foot
